arches/app/media/js/bindings/term-search.js

```diff
@@ -28,6 +28,10 @@ define([
                 }); 
                 searchbox.trigger('change');
             });
```

Comment:
If you add both a resource-type filter along with a term-filter and hit the clear button you'll notice that you'll need this logic here too:

```suggestion
                if (terms.length == 0 && searchbox.tags.length == 0){
                    searchbox.empty();
                }
```

Reply:
I'm assuming this committed change was limited to line 29 as otherwise the deleted `});` breaks things